packages/frontend/src/context/Post.ts

```diff
@@ -88,10 +88,11 @@ export class Data {
     }
 
     async loadFeed(query: string, lastRead = '0', epks = [] as string[]) {
+        const epksBase10 = epks.map((epk) => Number('0x' + epk))
```

Comment:
It's not safe to use a `Number` here if we have epoch keys larger than 64 bits. Maybe use `BigInt` instead?